EAD/EADSession2/Scripts/validation.js

```diff
@@ -0,0 +1,111 @@
+function validation(){
+	if(!nameValidation()){
+		return false;
+	} else if(!emailValidation(document.signupform)){
```

Comment:
we are passing document.signupform to a method. Ideally validation methods accept input value and return boolean output.